samples/HttpClient_Instapush/component.mk

```diff
@@ -1,2 +1,3 @@
+HWCONFIG := spiffs
 DISABLE_SPIFFS = 1
```

Comment:
This sample is using spiffs - just remove line 2. The original component.mk was wrong.

Reply:
Another way to look at this is:

```
HWCONFIG := spiffs    # I want a SPIFFS partition please
DISABLE_SPIFFS = 1    # BUT I don't want any default filesystem image (i.e. leave it un-initialised)
```